doc/src/manual/types.md

```diff
@@ -1332,46 +1331,45 @@ julia> get(Nullable(1.0), 0.0)
 ```
 
 !!! tip
-    Make sure the type of the default value passed to [`get()`](@ref) and that of the [`Nullable`](@ref)
+    Make sure the type of the default value passed to `get()` and that of the `Nullable`
     object match to avoid type instability, which could hurt performance. Use [`convert()`](@ref)
     manually if needed.
 
-### Performing operations on [`Nullable`](@ref) objects
+### Performing operations on `Nullable` objects
 
-[`Nullable`](@ref) objects represent values that are possibly missing, and it
+`Nullable` objects represent values that are possibly missing, and it
 is possible to write all code using these objects by first testing to see if
 the value is missing with [`isnull()`](@ref), and then doing an appropriate
 action. However, there are some common use cases where the code could be more
 concise or clear by using a higher-order function.
 
-The [`map`](@ref) function takes as arguments a function `f` and a
-[`Nullable`](@ref) value `x`. It produces a [`Nullable`](@ref):
+The [`map`](@ref) function takes as arguments a function `f` and a `Nullable` value
+`x`. It produces a `Nullable`:
 
  - If `x` is a missing value, then it produces a missing value;
- - If `x` has a value, then it produces a [`Nullable`](@ref) containing
+ - If `x` has a value, then it produces a `Nullable` containing
    `f(get(x))` as value.
 
 This is useful for performing simple operations on values that might be missing
 if the desired behaviour is to simply propagate the missing values forward.
 
 The [`filter`](@ref) function takes as arguments a predicate function `p`
-(that is, a function returning a boolean) and a [`Nullable`](@ref) value `x`.
-It produces a [`Nullable`](@ref) value:
+(that is, a function returning a boolean) and a `Nullable` value `x`.
+It produces a `Nullable` value:
 
  - If `x` is a missing value, then it produces a missing value;
  - If `p(get(x))` is true, then it produces the original value `x`;
  - If `p(get(x))` is false, then it produces a missing value.
 
-In this way, [`filter`](@ref) can be thought of as selecting only allowable
+In this way, `filter` can be thought of as selecting only allowable
 values, and converting non-allowable values to missing values.
 
-While [`map`](@ref) and [`filter`](@ref) are useful in specific cases, by far
-the most useful higher-order function is [`broadcast`](@ref), which can handle
-a wide variety of cases.
+While `map` and `filter` are useful in specific cases, by far the most useful
+higher-order function is [`broadcast`](@ref), which can handle a wide variety of cases.
 
-[`broadcast`](@ref) can be thought of as a way to make existing operations work
+`broadcast` can be thought of as a way to make existing operations work
```

Comment:
You mean "`broadcast` can be thought of as a way to make existing operations work simultaneously and propagate nulls"? What does that mean?